models/user.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+const validator = require('validator');
+
+const userSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+    minlength: 2,
+    maxlength: 30,
+  },
+  about: {
+    type: String,
+    required: true,
+    minlength: 2,
+    maxlength: 30,
+  },
+  avatar: {
+    type: String,
+    required: true,
+    validate: {
+      validator: (item) => {
+        validator.isURL(item);
+      },
```

Comment:
Нужно исправить: валидация поля не работает.
Согласно документации ожидается, что метод валидации вернёт `true/false`. Это и будет означать, прошла валидация или нет.
В данном случае нет возврата значения, тк стрелочная функция автоматически возвращает значение только в случае короткой записи (иначе ожидается, что будет использована конструкция `return`):
```suggestion
      validator: (item) => validator.isURL(item),
```